src/components/EffectiveDateTime/EffectiveDateTime.vue

```diff
@@ -0,0 +1,477 @@
+<template>
+  <v-card flat id="effective-date-time-box">
+    <v-radio-group column class="pt-0 mt-0" v-model="effectiveDateType">
+      <v-radio label="Immediate (date and time of filing)" :value=EffectiveDateTypes.IMMEDIATE />
+      <v-radio label="A date and time in the future" :value=EffectiveDateTypes.FUTURE_EFFECTIVE />
+    </v-radio-group>
+
+    <v-form ref="form" class="date-time-selectors">
+      <DatePicker
+        ref="datePickerRef"
+        title="Date"
+        nudge-right="40"
+        :inputRules="dateRules"
+        :disablePicker="effectiveDateType !== EffectiveDateTypes.FUTURE_EFFECTIVE"
+        :minDate="dateToYyyyMmDd(minDate)"
+        :maxDate="dateToYyyyMmDd(maxDate)"
+        @emitDate="dateText = $event"
+        @emitCancel="dateText = ''"
+      />
+
+      <v-row>
+        <v-col cols="12" sm="6" md="3">
+          <v-combobox
+            id="hour-selector"
+            ref="hourSelector"
+            filled
+            class="mr-1"
+            v-model="selectHour"
+            label="Hour"
+            :items="hours"
+            :disabled="!isFutureEffective"
+            :rules="hourRules" />
+        </v-col>
+        <span class="time-colon" :class="{ 'disabled': !isFutureEffective }">:</span>
+        <v-col cols="12" sm="6" md="3">
+          <v-combobox
+            id="minute-selector"
+            ref="minuteSelector"
+            filled
+            class="ml-1"
+            v-model="selectMinute"
+            label="Minute"
+            :items="minutes"
+            :disabled="!isFutureEffective"
+            :rules="minuteRules" />
+        </v-col>
+        <v-col cols="12" sm="6" md="3">
+          <v-select
+            id="period-selector"
+            filled
+            v-model="selectPeriod"
+            :items="timePeriod"
+            :disabled="!isFutureEffective" />
+        </v-col>
+        <v-col cols="12" sm="6" md="3" class="label-col">
+          <span class="time-zone-label" :class="{ 'disabled': !isFutureEffective }">Pacific time</span>
+        </v-col>
+      </v-row>
+
+      <!-- display validation alert only after date and time have been entered -->
+      <v-row v-if="isFutureEffective && dateText && (selectHour.length > 0) && (selectMinute.length > 0)">
+        <v-col class="validation-alert">
+          <p class="validation-alert-msg" v-if="isUnderTime">
+            The time must be at least {{ dateToPacificTime(minDate) }} for the selected date
+          </p>
+          <p class="validation-alert-msg" v-if="isOverTime">
+            The time must be at most {{ dateToPacificTime(maxDate) }} for the selected date
+          </p>
+        </v-col>
+      </v-row>
+    </v-form>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Mixins, Prop, Vue, Watch } from 'vue-property-decorator'
+import { DatePicker } from '@/components/DatePicker' // NB: local component
```

Comment:
Ideally we would import the bcrs-shared-component DatePicker here, but I was getting Storybook errors here (while everything else worked).
I killed a few hours on this so ... this is what I'm going with atm.

Reply:
I updated this. See comment on `src/components/index.ts` below.

TLDR: this line is now more obviously a "local import" and it looks the same in the parent app; but the `components/index.ts` file will reference either the local file or the external file, respectively. This keeps things generic/consistent as much as possible while still providing the needed references.